<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="VIGNESH-DESKTOP-M7BTJCA" failures="1" tests="2" name="Default test" time="15.674" errors="0" timestamp="2023-10-22T19:19:28 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.873"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.291">
    <failure type="java.lang.AssertionError" message="expected [] but found [Username and password do not match any user in this service]">
      <![CDATA[java.lang.AssertionError: expected [] but found [Username and password do not match any user in this service]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefanitions.loginStepDefanition.errorMessage(loginStepDefanition.java:75)
at âœ½.User get error message(file:///C:/Users/vigne/eclipse-workspace/cucumber/cucumberPract/src/test/resources/featurefile/login.feature:16)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
